presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplitManager.java

```diff
@@ -143,6 +133,41 @@ public ConnectorSplitSource getSplits(
         }
     }
 
+    private static long[] getOffsets(KafkaConsumer<ByteBuffer, ByteBuffer> consumer, PartitionInfo partition, long startTimestamp, long endTimestamp)
```

Comment:
would it better return a pair of long? instead of having a long array?
Or, we could inline this functions

Reply:
Good point. Inlined, fixed.